### DISCUSSION TITLE
为什么PaddleOCR中的train和eval对图片的预处理不是同一个步骤，甚至输入模型的图片尺寸都不一样？

### DISCUSSION THREAD

**Discussion by lazyboooooy** [PaddlePaddle/PaddleOCR#13795] [Q&A]
因为我要将模型部署到RK3568上，所以要固定模型尺寸，但是我训练的时候打印尺寸发现训练和验证的尺寸不同，于是我追踪对图片的预处理，发现训练时对图片的预处理为:
<ppocr.data.imaug.operators.DecodeImage object at 0x7fc8fb052180>
<ppocr.data.imaug.label_ops.DetLabelEncode object at 0x7fc8fb052330>
<ppocr.data.imaug.iaa_augment.IaaAugment object at 0x7fc8fb053800>
<ppocr.data.imaug.random_crop_data.EastRandomCropData object at 0x7fc8fc0825d0>
<ppocr.data.imaug.make_border_map.MakeBorderMap object at 0x7fc907341f70>
<ppocr.data.imaug.make_shrink_map.MakeShrinkMap object at 0x7fc8fb8aa510>
<ppocr.data.imaug.operators.NormalizeImage object at 0x7fc8fb441eb0>
<ppocr.data.imaug.operators.ToCHWImage object at 0x7fc8fb443890>
<ppocr.data.imaug.operators.KeepKeys object at 0x7fc8fb8a9bb0>
而验证的时候对图片的预处理为:
<ppocr.data.imaug.operators.DecodeImage object at 0x7f41e9700740>
<ppocr.data.imaug.label_ops.DetLabelEncode object at 0x7f41f587cc20>
<ppocr.data.imaug.operators.DetResizeForTest object at 0x7f41e97006e0>
<ppocr.data.imaug.operators.NormalizeImage object at 0x7f41e9b49010>
<ppocr.data.imaug.operators.ToCHWImage object at 0x7f41ea80b4a0>
<ppocr.data.imaug.operators.KeepKeys object at 0x7f41eb294e60>
Q1.训练和验证的尺寸不同会对模型的精度有影响吗？如果有影响，影响固定尺寸前模型的精度，还是影响固定尺寸后模型的精度
Q2.如果对训练模型的精度有影响，可不可以将训练和验证的尺寸统一，从而降低对固定模型尺寸对模型精度造成的影响

**Comment by lazyboooooy** [comment]
补充一下:训练的模型为ch_PP-OCRv3_det Paddle版本为2.6.1

**Comment by GreatV** [comment]
训练时，图像的预处理通常包含各种数据增强技术，例如随机裁剪、旋转、颜色调整等，以便让模型在不同情况下都能表现良好。而验证时的预处理通常更简单，仅包括解码、归一化和尺寸调整，目的是保证验证结果的一致性和可重复性。

**Reply by GreatV** [reply]
你可能需要自己尝试一下让两者保持一致，看看效果。

**Reply by lazyboooooy** [reply]
ok 我试一下
